Stellar-transaction.x

```diff
@@ -578,6 +579,22 @@ struct InvokeHostFunctionOp
     SorobanAuthorizationEntry auth<>;
 };
 
+enum BumpExpirationType
```

Comment:
What would a "non-uniform" rent bump look like? Does uniform here mean that every key is bumped by the same amount of ledgers?